frontend/fingertips-frontend/components/pages/chart/index.tsx

```diff
@@ -0,0 +1,29 @@
+'use client';
```

Comment:
Can we move this `use client` directive down into the `LineChart` component? As I think it's the `HighChartsReact` component that requires us to use it, isn't it?